Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -651,6 +651,11 @@ RefPtr<StyleRuleKeyframes> CSSParserImpl::consumeKeyframesRule(CSSParserTokenRan
     AtomString name;
     if (nameToken.type() == IdentToken) {
         name = nameToken.value().toAtomString();
+        // From the CSS Values specification, keyframe names are not allowed to be
+        // CSS wide keywords. Nor can they be "default". And CSS Animations additionally
+        // excludes the "none" keyword.
+        if (CSSParserFastPaths::isUniversalKeyword(name) || equalLettersIgnoringASCIICase(name, "default"_s) || equalLettersIgnoringASCIICase(name, "none"_s))
```

Comment:
I think this should work and align more closely with spec words (and you won't need to do changes to `isUniversalKeyword`):

```suggestion
        if (isValidCustomIdentifier(nameToken.id()) || nameToken.id() == CSSValueNone)
```